redbot/cogs/customcom/customcom.py

```diff
@@ -202,6 +203,30 @@ def __init__(self, bot):
         """Custom commands management."""
         pass
 
+    @customcom.command(name="search")
+    @commands.guild_only()
+    async def cc_serach(self, ctx: commands.Context, *, search):
```

Comment:
spelling issue :D

Reply:
Yeah, thanks for pointing it out.  I'll be pushing a commit in a second to address that.